src/pages/guide/css/preprocess.md

```diff
@@ -263,6 +263,49 @@ In the processed file, this results in the following:
  // Theme widgets
 ```
 
+## Trigger flag deploy styles for only enabled modules
+
+Default all css by default will build to final css no matter modules enabled or disabled. With this flag explain below allow developers able to control how we want css generate
+
+
+### Changing flag trigger deploy static contents
+
+1. Add inside to app/etc/env.php or app/etc/config.php. Add following line to the top array
+
+```php
+return [
+'static_content_only_enabled_modules' => true,
+//...other lines
+]
+```
+
+With true value meaning magento will deploy static content styles from enabled module to final css files (styles-l.css, styles-m.css). If set value to false magento will deploy all styles no matter modules enabled or disabled!
+
+
+2. After modify php file. Make sure re-run setup upgrade for update config flag 
+
+
+sync-up config
+
+```bash
+php bin/magento app:config:import
+```
+
+and run setup command
+
+```bash
+php bin/magento setup:upgrade
```

Comment:
```suggestion
bin/magento setup:upgrade
```

Reply:
Please increase the indent of the code block 3 spaces so that it lines up under the previous step.